plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/metastore/glue/TestDeltaLakeRenameToWithGlueMetastore.java

```diff
@@ -0,0 +1,139 @@
+
+import java.io.File;
+import java.util.Optional;
+
+import static com.google.common.util.concurrent.MoreExecutors.directExecutor;
+import static io.trino.plugin.deltalake.DeltaLakeConnectorFactory.CONNECTOR_NAME;
+import static io.trino.testing.TestingSession.testSessionBuilder;
+import static io.trino.testing.sql.TestTable.randomTableSuffix;
+import static java.lang.String.format;
+
+public class TestDeltaLakeRenameToWithGlueMetastore
+        extends AbstractTestQueryFramework
+{
+    protected static final String SCHEMA = "test_delta_lake_rename_to_with_glue_" + randomTableSuffix();
+    protected static final String CATALOG_NAME = "delta_lake_rename_to_with_glue";
+    protected File metastoreDir;
+    protected HiveMetastore metastore;
+    protected HdfsEnvironment hdfsEnvironment;
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        Session deltaLakeSession = testSessionBuilder()
+                .setCatalog(CATALOG_NAME)
+                .setSchema(SCHEMA)
+                .build();
+
+        DistributedQueryRunner queryRunner = DistributedQueryRunner.builder(deltaLakeSession).build();
+
+        this.metastoreDir = new File(queryRunner.getCoordinator().getBaseDataDir().resolve("delta_lake_data").toString());
+        this.metastoreDir.deleteOnExit();
+
+        queryRunner.installPlugin(new DeltaLakePlugin());
+        queryRunner.createCatalog(
+                CATALOG_NAME,
+                CONNECTOR_NAME,
+                ImmutableMap.<String, String>builder()
+                        .put("hive.metastore", "glue")
+                        .put("hive.metastore.glue.region", "us-east-2")
+                        .put("hive.metastore.glue.default-warehouse-dir", metastoreDir.getPath())
+                        .buildOrThrow());
+
+        HdfsConfig hdfsConfig = new HdfsConfig();
+        hdfsEnvironment = new HdfsEnvironment(
+                new DynamicHdfsConfiguration(new HdfsConfigurationInitializer(hdfsConfig), ImmutableSet.of()),
+                hdfsConfig,
+                new NoHdfsAuthentication());
+        GlueHiveMetastoreConfig glueConfig = new GlueHiveMetastoreConfig()
+                .setGlueRegion("us-east-2");
+        metastore = new GlueHiveMetastore(
+                hdfsEnvironment,
+                glueConfig,
+                DefaultAWSCredentialsProviderChain.getInstance(),
+                directExecutor(),
+                new DefaultGlueColumnStatisticsProviderFactory(directExecutor(), directExecutor()),
+                Optional.empty(),
+                table -> true);
+
+        queryRunner.execute("CREATE SCHEMA " + SCHEMA + " WITH (location = '" + metastoreDir.getPath() + "')");
+        return queryRunner;
+    }
+
+    @Test
+    public void testRenameOfExternalTable()
+    {
+        String oldTable = "table_external_to_be_renamed_" + randomTableSuffix();
+        String newTable = "table_external_renamed_" + randomTableSuffix();
```

Comment:
Lets' prepend `test_` as `TestDatabricksWithGlueMetastoreCleanUp` expects the prefix.